streams/src/main/java/org/apache/kafka/streams/kstream/TumblingWindows.java

```diff
@@ -23,12 +23,13 @@
 import java.util.Map;
 
 /**
- * The tumbling window specifications used for aggregations.
+ * The tumbling window specifications used for aggregations. TODO extend
```

Comment:
We may also need to explain:

1. the role/impact of (a) the emit behavior and, closely related, (b) the semantics of the timestamp extractor (think: "emits are always triggered based on wall clock time, but windows -- and how records are assigned to windows -- are always based on the configured timestamp extractor, which means it could be wall clock time, producer time, or ...).  Same for the docstring of `HoppingWindows`.